app/assets/stylesheets/components/buttons.scss

```diff
@@ -261,6 +265,18 @@
   --color-inverted-hover: var(--brand-github-color);
 }
 
+.crayons-btn--brand-facebook {
+  --bg: var(--brand-facebook-bg);
+  --bg-hover: var(--brand-facebook-bg-hover);
+  --color: var(--brand-facebook-color);
+  --color-hover: var(--brand-facebook-color);
+  --bg-inverted: var(--brand-facebook-bg);
+  --bg-inverted-hover: var(--brand-facebook-bg-hover);
+  --color-inverted: var(--brand-facebook-color);
+  --color-inverted-hover: var(--brand-facebook-color);
```

Comment:
@ludwiczakpawel they actually do belong here. I noticed in doing this that Facebook's styles aren't described anywhere so that button defaults to brand color. I think that went unnoticed because our blue-purple button is decently close to Facebook brand blue.

Reply:
gotcha.. well, then good catch! :)